x-pack/plugins/security_solution/public/detections/components/callouts/documentation_links.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { ExternalLink } from './external_link';
+
+export const SecuritySolutionRequirementsLink = () => (
+  <ExternalLink
+    url="https://www.elastic.co/guide/en/security/current/sec-requirements.html"
+    text="Elastic Security system requirements"
```

Comment:
I think that's a good point to discuss! Like I mentioned in the PR description

> You can notice that callout messages are localized, but links to documentation are not. I did it on purpose to keep it simple and because I'm not sure they have to be localizable. The reason is - I'm not even sure we have docs for security solution in other languages. For me it seems like docs in other languages are outdated. Pls correct me if I'm wrong.

Options:

1. Keeps it as is = links to docs are not localized.
2. Localize link text, but keep the url to the EN documentation.
3. Localize both link text and url, so localizers could point to a different (translated) article.

If translated documentation for Elastic Security exists or there are plans to write it, 3 is the way to go. If there are no plans to build or maintain translated docs, let's do 1. I'm not sure if 2 makes sense.

What do you think?

Reply:
So we don't currently have any security docs in a language other than English. There are some CN, JP, and KR docs available [here](https://www.elastic.co/guide/index.html#_docs_in_your_native_tongue), but nothing for Elastic Security, so that's a no-go for option 3. Looks like we've historically gone with option 2 [elsewhere in the app](https://github.com/elastic/kibana/blob/2f386e89619488c97859db2ec5f57ebe026ecb91/x-pack/plugins/security_solution/public/network/components/embeddables/embedded_map.tsx#L227), so I would say let's follow suit with that until we have locale specific documentation. Then at least the in-app copy will be consistent with the user's current locale. 

Note, there are variables for  the elastic domain and doc version, so we'll want to use those within the URL. The [above example](https://github.com/elastic/kibana/blob/2f386e89619488c97859db2ec5f57ebe026ecb91/x-pack/plugins/security_solution/public/network/components/embeddables/embedded_map.tsx#L224) uses this method,  but it's as simple as pulling in kibana services and referencing the `docLinks`:


``` ts
const { services } = useKibana();
```
``` jsx
<EuiLink
  href={`${services.docLinks.ELASTIC_WEBSITE_URL}guide/en/security/${services.docLinks.DOC_LINK_VERSION}/conf-map-ui.html`}
  target="_blank"
>
  {i18n.EMBEDDABLE_HEADER_HELP}
```

Reply:
I'll address it in this PR. Thank you for pointing to `services.docLinks`, great catch 👍